plugins/view-transitions/includes/settings.php

```diff
@@ -346,12 +359,33 @@ function plvt_render_settings_field( array $args ): void {
 			?>
 		</select>
 		<?php
+	} elseif ( 'checkbox' === $type ) {
+		?>
+		<label>
+			<input
+				id="<?php echo esc_attr( $args['label_for'] ); ?>"
```

Comment:
The code above adds `label_for` which means the label is rendered on the left column automatically. If now the label is rendered here, there would be a duplicate label. I think it makes sense to have it here, so maybe you just have to not pass `label_for` above in case of a checkbox.